src/transformStateWithClones.js

```diff
@@ -7,7 +7,42 @@
  * @return {Object[]}
  */
 function transformStateWithClones(state, actions) {
-  // write code here
+  const result = [];
+  const stateCopy = { ...state };
+
+  for (let i = 0; i < actions.length; i++) {
+    const action = actions[i];
```

Comment:
you may use a common loop here for sure, but it'll look a bit better with `for of`